app/workers/statsd_census_job.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+##
+# This job should be scheduled to run every 30m or so. Its purpose is to ensure all StatsD.increment metrics are
+# initialized with a StatsD.increment(stat_key, 0) so they can properly be captured by Grafana#
+class StatsDCensusJob
+  KNOWN_STATSD_TAGS = [].freeze
+
+  include Sidekiq::Worker
+  sidekiq_options(retry: false)
+
+  def perform
+    initialize_stats_roster unless stats_roster_is_in_redis?
+    audit_roster!
+  end
+
+  private
+
+  def initialize_stats_roster
+    Redis.current.set(stats_roster_key, 1)
+  end
+
+  def stats_roster_is_in_redis?
```

Comment:
Is this where we want to grab our SET of statsD keys from redis or check if the SET exists?